vm/class_test.go

```diff
@@ -990,3 +990,24 @@ func TestClassSingletonClassClassMethod(t *testing.T) {
 		v.checkSP(t, i, 1)
 	}
 }
+
+func TestObjectId(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected interface{}
+	}{
+		{`a = nil.object_id; b = nil.object_id; a == b`, true},
+		{`a = true.object_id; b = true.object_id; a == b`, true},
+		{`a = false.object_id; b = false.object_id; a == b`, true},
+		{`a = 1.object_id; b = 1.object_id; a == b`, false},
+		{`a = "a".object_id; b = "a".object_id; a == b`, false},
```

Comment:
There are serveral built-in classes which have the literal syntax, Should we have test cases for them?
e.g. `24..7`, `{}`, `[]`, `:sym`
https://github.com/goby-lang/goby/blob/cc8f18446d36d19945f97a1de9000f79c9bc6558/compiler/parser/data_type_parsing.go

Reply:
Added. Thank you!